allennlp/models/basic_classifier.py

```diff
@@ -163,6 +163,17 @@ def make_output_human_readable(
             )
             classes.append(label_str)
         output_dict["label"] = classes
+        tokens = []
+        for instance_tokens in output_dict["token_ids"]:
+            tokens.append(
+                [
+                    self.vocab.get_token_from_index(
+                        token_id.item(),namespace="tags"
```

Comment:
Yeah, this looks like the right approach.  The only problem is this namespace.  We don't want to hard-code this here, but let it be specified in the model constructor.  Can you add this as a parameter above?

This is also why the tests are failing, which you can see here: http://build.allennlp.org/viewLog.html?buildId=28680&buildTypeId=AllenNLP_AllenNLPPullRequests&tab=buildLog (log in as guest).  The default value for the constructor parameter should be whatever makes those tests pass, which is probably "tokens".

Reply:
I added the namespace to the model constructor. But the test still fails? Can you double-check I understand you correctly?